embulk-input-postgresql/src/main/java/org/embulk/input/PostgreSQLInputPlugin.java

```diff
@@ -53,6 +53,10 @@
         @Config("application_name")
         @ConfigDefault("\"embulk-input-postgresql\"")
         public String getApplicationName();
+
+        @Config("statement_timeout")
+        @ConfigDefault("-1")
+        public int getStatementTimeout();
```

Comment:
`-1` to disable (by default) seems like a traditional manner. How about:

```
@Config("statement_timeout")
@ConfigDefault("null")
public Optional<Integer> getStatementTimeout();
```

Even if we use a number to disable, `0` won't work?

----

At the same time, I was at first concerned about its unit (seconds or milliseconds), and thought to name it `statement_timeout_millis` or like that.

But, for the consistency with the existing `connect_timeout` and `socket_timeout` (they look to be milliseconds), `statement_timeout` would be good.

Reply:
if we set it to 0 it means no timeout.

Thanks for your suggestion.